### PR TITLE
Health api for metamorph

### PR DESCRIPTION
**Pull request by shotasilagadzetaal**

### CODE REVIEW

File: api/handler/default.go [hunk 1]
```diff
@@ -80,6 +80,29 @@ func (m ArcDefaultHandler) GETPolicy(ctx echo.Context) error {
...
+	healthy := true
+
+	err := m.TransactionHandler.Health(ct)
+
+	if err == nil {
+		return ctx.JSON(http.StatusOK, api.Health{
+			Healthy: &healthy,
+			Reason:  nil,
+		})
+	} else {
```
Comment (boecklim) [inline api/handler/default.go:96]: The `else` is not needed here

File: api/handler/default.go [hunk 2]
```diff
@@ -80,6 +80,29 @@ func (m ArcDefaultHandler) GETPolicy(ctx echo.Context) error {
...
+		healthy = false
+		reason := err.Error()
+		return ctx.JSON(http.StatusOK, api.Health{
+			Healthy: &healthy,
```
Comment (boecklim) [inline api/handler/default.go:100]: Consider using `PtrTo(false)` and `PtrTo(err.Error())` here

File: api/handler/default.go [hunk 3]
```diff
@@ -80,6 +80,29 @@ func (m ArcDefaultHandler) GETPolicy(ctx echo.Context) error {
 	})
 }
 
+func (m ArcDefaultHandler) GETHealth(ctx echo.Context) error {
```
Comment (boecklim) [inline api/handler/default.go:83]: Can we have a unit test for this function?

File: api/handler/default.go (hunk 2 above)
Comment (shotasilagadzetaal) [reply api/handler/default.go:100]: `PtrTo(err.Error())` doesn't work :/ I fixed `false`

**Review by boecklim — APPROVED** [review]

**Pull request merged**